terminus/models/city.py

```diff
@@ -44,6 +49,22 @@ def add_intersection_at(self, point):
                 if road.includes_point(point):
                     road.replace_node_at(point, intersection)
 
+    def bounding_box(self):
+        road_boxes = self._box_list_from_list(self.roads)
```

Comment:
Although it is a slight variation, you can do `self._bounding_box_from_list(self.roads)`, where:

```
def _box_list_from_list(self, list):
    return BoundingBox.from_boxes(map(lambda x: x.bounding_box(), list))
```

this will save you a couple of lines of code.